### DISCUSSION TITLE
Somatic CNV calling in case of pre-existing CNVs

### DISCUSSION THREAD

**Discussion by HenriettaHolze** [etal/cnvkit#1013] [General]
Hi, could you explain how reliable CNVkit is for somatic CNV calling in case of "germline" CNVs?   
We sequenced a cancer cell line before and after engraftment into mice. I ran CNVkit with the nf-core/sarek pipeline. 
When looking at the results for the baseline sample, CNVkit shows a locus with 5 copies at baseline (~2.4M bp).  
We are interested in additional CNVs that happened/were selected for during engraftment, i.e. further amplification or loss.  
The results of CNVkit and the PURPLE tool (run as part of nf-core/oncoanalyser pipeline) are quite different: CNVkit estimates a somatic CN change of 1.38 (cn=3, log2=0.456329) at the locus. The CN estimated by purple is 2.03.

Below a scatter plot of chr12 for the baseline sample
![image](https://github.com/user-attachments/assets/668b2d14-5a63-426e-94fc-e2cc6c02ec03)

And for the tumor sample using the baseline as reference
![image](https://github.com/user-attachments/assets/4838be04-08c8-4392-a35c-65b29c972eb1)

Thanks!

**Comment by etal** [comment]
Good question. It depends on the size of the "normal" CNV and whether the reference is "flat" or paired/pooled from the original cell line.

I recall PURPLE uses XHMM but I'm not sure how oncoanalyzer constructs the PoN reference, or if there is one. That would make all the difference in your case.

In any case, once you find out which it is, you can use CNVkit with a flat, paired, or pooled reference to try to match what oncoanalyzer is doing. At that point, if any discrepancies remain, we can drill into other behavior specific to CNVkit.
